framework/src/arastorage/relation.c

```diff
@@ -1023,6 +1063,9 @@ db_result_t relation_process_remove(db_handle_t **handle, db_cursor_t *cursor)
 		goto errout;
 	}
 	memcpy((*handle)->result_rel->name, (*handle)->rel->name, sizeof((*handle)->result_rel->name));
+	if (storage_get_row_amount((*handle)->result_rel, &nrows) == DB_OK) {
```

Comment:
Do you use storage_get_row_amount to verify to write well? It not, result_rel->cardinality is enough.

Reply:
use to do lseek, avoid occasional row amout reading issue after save file.